src/Shared/Cosmos/CosmosConstants.cs

```diff
@@ -5,6 +5,12 @@ namespace Aspire.Hosting.Azure.Cosmos;
 
 internal static class CosmosConstants
 {
+    /// <summary>
+    /// Defines the application name used to interact with the Azure cosmos db. This will be suffixed to the
+    /// cosmos user-agent to include with every Azure Cosmos DB service interaction.
+    /// </summary>
```

Comment:
```suggestion
    /// <summary>
    /// Defines the application name used to interact with the Azure Cosmos database. This will be suffixed to the
    /// Cosmos user-agent to include with every Azure Cosmos database service interaction.
    /// </summary>
```
@Pilchie would this be a correct change?

Reply:
LGTM @RussKie

Reply:
I'm going to apply the same to the main